<ui version="4.0" >
 <class>KexiConnSelector</class>
 <widget class="QWidget" name="KexiConnSelector" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>370</width>
    <height>191</height>
   </rect>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" >
    <widget class="QLabel" name="lblIcon" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Preferred" hsizetype="Fixed" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text" >
      <string/>
     </property>
     <property name="alignment" >
      <set>Qt::AlignTop</set>
     </property>
     <property name="wordWrap" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="5" >
    <widget class="QLabel" name="label" >
     <property name="text" >
      <string>&lt;b>Select Existing Database Server's Connection From the List Below&lt;/b>
&lt;p>You will see existing Kexi projects available for the selected connection. Here you may also add, edit or remove connections from the list.
</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignTop</set>
     </property>
     <property name="wordWrap" >
      <bool>true</bool>
     </property>
     <property name="buddy" >
      <cstring>list</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="6" >
    <widget class="Q3ListView" name="list" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="MinimumExpanding" hsizetype="Preferred" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize" >
      <size>
       <width>0</width>
       <height>60</height>
      </size>
     </property>
     <property name="selectionMode" >
      <enum>Q3ListView::Single</enum>
     </property>
     <property name="allColumnsShowFocus" >
      <bool>true</bool>
     </property>
     <property name="showSortIndicator" >
      <bool>true</bool>
     </property>
     <property name="resizeMode" >
      <enum>Q3ListView::LastColumn</enum>
     </property>
     <column>
      <property name="text" >
       <string>Name</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>Type</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>Server Information</string>
      </property>
     </column>
    </widget>
   </item>
   <item rowspan="2" row="2" column="0" colspan="2" >
    <widget class="QGroupBox" name="descGroupBox" >
     <property name="title" >
      <string>Description</string>
     </property>
     <layout class="QGridLayout" >
      <property name="leftMargin" >
       <number>6</number>
      </property>
      <property name="topMargin" >
       <number>6</number>
      </property>
      <property name="rightMargin" >
       <number>6</number>
      </property>
      <property name="bottomMargin" >
       <number>6</number>
      </property>
      <item row="0" column="0" >
       <widget class="QLabel" name="descriptionLabel" >
        <property name="focusPolicy" >
         <enum>Qt::StrongFocus</enum>
        </property>
        <property name="text" >
         <string/>
        </property>
        <property name="alignment" >
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
        </property>
        <property name="wordWrap" >
         <bool>true</bool>
        </property>
        <property name="openExternalLinks" >
         <bool>true</bool>
        </property>
        <property name="textInteractionFlags" >
         <set>Qt::LinksAccessibleByKeyboard|Qt::LinksAccessibleByMouse|Qt::NoTextInteraction|Qt::TextBrowserInteraction|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="2" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Preferred</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>50</width>
       <height>29</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="3" >
    <widget class="QPushButton" name="btn_add" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text" >
      <string>&amp;Add...</string>
     </property>
    </widget>
   </item>
   <item row="2" column="4" >
    <widget class="QPushButton" name="btn_edit" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text" >
      <string>&amp;Edit...</string>
     </property>
    </widget>
   </item>
   <item row="2" column="5" >
    <widget class="QPushButton" name="btn_remove" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text" >
      <string>&amp;Remove</string>
     </property>
    </widget>
   </item>
   <item row="3" column="4" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Preferred</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>20</width>
       <height>41</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>Q3ListView</class>
   <extends>Q3Frame</extends>
   <header>q3listview.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>list</tabstop>
  <tabstop>btn_add</tabstop>
  <tabstop>btn_edit</tabstop>
  <tabstop>btn_remove</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
